components/buttons/ApplyJob.tsx

```diff
@@ -0,0 +1,28 @@
+import { IJob } from '@/types/job';
```

Comment:
@Shurtu-gal we have removed the whole jobs section so we don't need this file in the new branch

Reply:
Done